AGENTS.md

```diff
@@ -358,6 +358,52 @@ For optimal performance with DDEV development, consider these configuration patt
 }
 ```
 
+### Claude Code for Web Environment
+
+The web-based Claude Code environment has specific limitations compared to desktop environments:
+
+**Environment Limitations:**
+
+- **Docker NOT Available**: Docker is not installed and cannot be used in the web environment
+- **Integration Tests**: Most DDEV integration tests require Docker and will fail in this environment
+- **Container Operations**: No ability to build, run, or test Docker containers
+
+**SessionStart Hook Solution:**
+
+A `.claude/SessionStart` hook automatically configures the environment for optimal web development:
```

Comment:
Is the hook automatically created in the Web environment or is something that should exist in the repo?